src/builders.ts

```diff
@@ -389,14 +403,19 @@ export class BomBuilder {
       return undefined
     }
 
+    // attention: `isDevOptional` does not cause a component to be optional, since it is still required for development
```

Comment:
:rotating_light: arguable.

i guess `isDevOptional` shoud lalso flagged as `optional` scoped, to indicate it has not other porpose.
If the property `development` is true, then it is on the reader to evaluate a possible scope change, since it is the inthe authors intention, not in the state's

Reply:
done